src/prefabs/structures/TimePicker/index.ts

```diff
@@ -0,0 +1,109 @@
+import { component, PrefabReference } from '@betty-blocks/component-sdk';
```

Comment:
> Why is this whole TimePicker structure added?
> 
> The DateTimePicker is capable to be used for: date, datetime and time selection.
> 
> There is also not a DatePicker structure, just a DateTimePicker structure. So I think this TimePicker structure is not necessary, and should be removed.

This is due to it not having disableFuture or the disablePast option. In the current prefab, timepicker has disablePast, but it doesn't apply anything. Due to it not doing anything, it shouldn't be included in the same prefab as datetime or date. The code can be the same because it ignores the flags.

Why this is in the same PR is because adding the checkbox disableFuture also adds it to the timepicker, which also doesn't work in this way.